### PR TITLE
Fix Clamp Roughness, move parameter out of NEE box

### PR DESCRIPTION
**Pull request by usakhelo**
Fixes clamp roughness and moves checkbox to separate groupbox

### CODE REVIEW

File: src/appleseed.studio/mainwindow/renderingsettingswindow.cpp [hunk 1]
```diff
@@ -1018,10 +1018,16 @@ namespace
             connect(unlimited_ray_intensity, SIGNAL(toggled(bool)), max_ray_intensity, SLOT(setDisabled(bool)));
         }
 
-        void create_pt_advanced_nee_clamp_roughness_settings(QVBoxLayout* parent)
+        void create_pt_advanced_clamp_roughness_settings(QVBoxLayout* parent)
```
Comment (dictoon) [inline src/appleseed.studio/mainwindow/renderingsettingswindow.cpp]: Should be called something like `create_pt_advanced_optimization_settings()`.

File: src/appleseed/renderer/kernel/lighting/pathtracer.h [hunk 2]
```diff
@@ -683,7 +683,7 @@ bool PathTracer<PathVisitor, VolumeVisitor, Adjoint>::process_bounce(
             vertex.m_scattering_modes,
             sample);
 
-        next_ray.m_max_roughness = m_clamp_roughness ? 0.0f : sample.m_max_roughness;
+        next_ray.m_max_roughness = m_clamp_roughness ? sample.m_max_roughness : 0.0f;
```
Comment (dictoon) [inline src/appleseed/renderer/kernel/lighting/pathtracer.h]: Totally missed that in my previous review :)

**Comment by dictoon** [comment]
Thanks Sergo!

**Comment by usakhelo** [comment]
No problem Franz, I'm happy to help.

**Pull request merged**